src/heuristics.rs

```diff
@@ -186,11 +186,15 @@ async fn make_wildcard_request(
                 let ferox_response = FeroxResponse::from(response, false).await;
                 let url_len = get_url_path_length(&ferox_response.url());
                 let content_len = ferox_response.content_length();
+                let content_words = ferox_response.word_count();
+                let content_lines = ferox_response.line_count();
 
                 if !CONFIGURATION.quiet && !should_filter_response(&ferox_response) {
                     let msg = format!(
-                        "{} {:>10} Got {} for {} (url length: {})\n",
+                        "{} {:>8} {:>8} {:>8} Got {} for {} (url length: {})\n",
```

Comment:
On lines 92 and 113 of this file, the final message looks like this:

`WLD       3526 Wildcard response is static; auto-filtering 3526 responses; toggle this behavior by using --dont-filter`

They should probably have the same column formatting as the others. (i can't add a comment on the exact lines for w/e reason)